.eslintignore

```diff
@@ -1,8 +1,8 @@
 # Common
-node_modules
-dist
-.nuxt
-coverage
+**/node_modules
+**/dist
+**/.nuxt
+**/coverage
 
 # Plugin
-lib/plugin.js
\ No newline at end of file
+**/plugin.js
```

Comment:
why did you change this?

Reply:
copied from other nuxt-community modules

Reply:
but here it makes no sense. I just want to ignore the linter on one file, not every file called `plugin.js`